acs_registry/__init__.py

```diff
@@ -25,6 +27,23 @@
 @agent.on_event("startup")
 async def introduce_agent(ctx: Context):
     ctx.logger.info(f"Agent: {agent.name} ({agent.address})")
+    ctx.storage.set("stations", [])
+
+
+@agent.on_interval(period=10)
```

Comment:
period will be variable on how long the ttl is. Is there any way to use a changeable variable which is set in the init function?